src/controllers/user.js

```diff
@@ -13,9 +16,8 @@ const signUp = async (req, res) => {
             familyName: user.familyName,
             dateOfBirth: user.dateOfBirth,
             address: user.address,
```

Comment:
Good job Manizha! the reusable function works well.  
I looked up your code and tested it on Postman, I think we don't need to return the address in this sign-up controller since it's been handled by the addressToCoordinate function, what do you think?

Reply:
Thanks for mentioning that Lara, that is good point.